packages/data-provider/src/controllers/submittedDataController.ts

```diff
@@ -0,0 +1,120 @@
+import { NextFunction, Request, Response } from 'express';
+import * as _ from 'lodash-es';
+
+import { BaseDependencies } from '../config/config.js';
+import submittedDataService from '../services/submittedDataService.js';
+import { BadRequest, NotFound } from '../utils/errors.js';
+import { isEmptyString } from '../utils/formatUtils.js';
+import { SubmittedDataPaginatedResponse } from '../utils/types.js';
+
+const controller = (dependencies: BaseDependencies) => {
+	const service = submittedDataService(dependencies);
+	const { logger } = dependencies;
+	const LOG_MODULE = 'SUBMITTED_DATA_CONTROLLER';
+	const defaultPage = 1;
+	const defaultPageSize = 20;
+	return {
+		getSubmittedDataByCategory: async (
+			req: Request<{ categoryId: string }, {}, {}, { pageSize: string; page: string }>,
+			res: Response,
+			next: NextFunction,
+		) => {
+			try {
+				const categoryId = Number(req.params.categoryId);
+				let page = parseInt(req.query.page as string) || defaultPage;
+				let pageSize = parseInt(req.query.pageSize as string) || defaultPageSize;
```

Comment:
considering this variables are not changing, what's preventing from these being defined as constants? (`let` vs `const`)